requirements.txt

```diff
@@ -2,5 +2,5 @@ flask
 Flask-Cors
 -e git+https://github.com/RedisGraph/redisgraph-py#egg=redisgraph
 -e git+https://github.com/HTTP-APIs/hydra-python-core#egg=hydra_python_core
--e git+https://github.com/HTTP-APIs/hydra-python-agent#egg=hydra_agent
+-e git://github.com/priyanshunayan/hydra-python-agent.git@get-request#egg=hydra_agent
```

Comment:
Please don't change this. Requirements should always point to the main repo.

Reply:
I can revert the change if this PR is merged: https://github.com/HTTP-APIs/hydra-python-agent/pull/149

Reply:
Reverted.